### ISSUE TITLE
Running tests against usegalaxy.be

### ISSUE THREAD

**Issue by ieguinoa** [usegalaxy-eu/workflow-testing#50]
would that be possible? :) 

I'm making sure the tools/dependencies for these wf are available. I guess you would need a specific user on out server... what else would you need? can I later add specific wf tests? or better yet add them to set to be tested on all servers (.eu  and .be for the moment)?

thanks!

**Comment by bgruening** [comment]
Yes, easily possible. But we need to invest some time in workflow testing again.

> what else would you need? 

API key of one user. We have create a `service-account`.

> can I later add specific wf tests? 

Please I beg you to do this! :)

> or better yet add them to set to be tested on all servers (.eu and .be for the moment)?

Yes, that would be the idea!

**Comment by ieguinoa** [comment]
ok, we have a deal :P 
I will work on the tests, can probably get some help from Bert to fix the trainings test if necessary.
And will get back to you so we can have a better set of test and add .be to it.
